prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -40,6 +40,20 @@ vm_job_template: &vm_job_template
             memory: 100Mi
             cpu: 50m
 
+vm_job_template_k3s: &vm_job_template_k3s
```

Comment:
This template is used only in serverless. Can you please change the name to something like `vm_job_template_serverless_k3s`?